# omp

Beta-barrel outer membrane proteins (OMP) are integral components of Gram-negative
bacteria and eukaryotic mitochondria, and chloroplasts. They play essential roles in various
cellular processes including nutrient transport, membrane stability, and host-pathogen
interactions, antibiotic resistance and more. The advent of AlphaFold2 for accurate protein
structure predictions transformed structural bioinformatic studies. We previously used a QTY
code to convert hydrophobic alpha-helices to hydrophilic alpha-helices in over 20 membrane
proteins with all alpha-helices. The QTY code systematically replaces hydrophobic leucine
(L), isoleucine (I), valine (V), and phenylalanine (F) to hydrophilic glutamine (Q), threonine
(T), and tyrosine (Y). We here present a structural bioinformatic analysis of five outer
membrane beta-barrel proteins with known molecular structures including: a) BamA, b)
Omp85, c) FecA, d) TSX, and e) OmpC. We superposed the structures of five native beta-
barrel outer membrane proteins and their AlphaFold2-predicted corresponding QTY variant
structures. The superposed structures of OMPs and their QTY variants exhibit remarkable
structural similarity, as evidenced by residue mean square distance (RMSD) values between
0.060Å to 0.456Å despite >44% transmembrane amino acid replacement. We also show that
native outer membrane proteins and QTY variants have different hydrophobicity patches. Our
study provides important insights into the differences between hydrophobic and hydrophilic
beta-barrels and validates the QTY code for studying beta-barrel membrane proteins and
perhaps other hydrophobic aggregated proteins. Our findings demonstrate the QTY code as a
simple tool for designing hydrophobic protein in various biological contexts.
